graph-db/src/utils/cmr/fetchPageFromCmr.js

```diff
@@ -0,0 +1,47 @@
+const fetch = require('node-fetch');
+const { getEchoToken } = require('./getEchoToken');
+
+/**
+ * fetchPageFromCMR: Fetch a page of collections from CMR
+ * search endpoint and initiate or continue scroll request
+ * @param scrollId {String} An optional scroll-id given from the CMR
+ * @returns [{JSON}] An array of UMM JSON collection results
+ */
+ exports.fetchPageFromCMR = async scrollId => {
+    const token = null; //getEchoToken();
```

Comment:
This is set to null for local testing purposes. This should be changed when in an operational environment

Reply:
Well, this should be accounted for, devs cant be expected to know where/what to comment out. You _could_ check the NODE_ENV environment variable or more reliably in a Serverless Framework app you can check the IS_OFFLINE environment variable within your `getEchoToken` method.